### PR TITLE
Data library sync bugfix

### LINKED ISSUE #5239: Check if downgrades blocks sync tables of being synced
If I'm not wrong, I remember that each sync period checks if the feature of sync tables is enabled. For a user that has some active sync tables and downgrades to a plan without this feature, the next syncing periods used to fail due to this.

P.S.: I'm testing this directly, I'll add the results in 1 hour.

### PR DESCRIPTION
**Pull request by Kartones**
Fixes #5239

### CODE REVIEW

**Comment by Kartones** [comment]
@juanignaciosl CR please (what a facepalm fail I did)

File: app/controllers/api/json/synchronizations_controller.rb [hunk 1]
```diff
@@ -132,6 +134,11 @@ def destroy
 
   private
 
+  def set_external_source
+    @external_source = params[:remote_visualization_id].present? ?
```
Comment (juanignaciosl) [inline app/controllers/api/json/synchronizations_controller.rb]: This line break because of the line length looks weird :-\

**Comment by juanignaciosl** [comment]
:+1:

File: app/controllers/api/json/synchronizations_controller.rb (hunk 1 above)
Comment (Kartones) [inline app/controllers/api/json/synchronizations_controller.rb]: yep... but don't know how to "cut" too long lines, in my Sublime they appear well but...

**Pull request merged**